.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,142 @@
+---
+name: E2E Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths-ignore:
+      - .circlecli/**
+      - meta/**
+      - .gitignore
+      - .markdownlint.yml
+      - .yamllint
+      - ansible.cfg.galaxy
+      - CHANGELOG.md
+      - LICENSE
+      - poetry.lock
+      - pyproject.toml
+      - README.md
+      - Vagrantfile
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+env:
+  PY_COLORS: true
+  ANSIBLE_FORCE_COLOR: true
+  MOLECULE_DEBUG: false
+  MOLECULE_VERBOSITY: 0
+  MOLECULE_NO_LOG: true
+
+concurrency:
+  group: e2e
+  cancel-in-progress: false
+
+jobs:
+  python36:
+    runs-on: ubuntu-20.04
+    strategy:
+      max-parallel: 1
+      fail-fast: false
+      matrix:
+        python:
+          - version: 3.6
+            ansible-core: 2.11.12
+            molecule: 3.6.1
+            molecule-docker: 1.1.0
```

Comment:
![Screenshot 2022-12-29 at 12-21-41 CI Use Ansible Molecule for end-to-end testing by mamercad · Pull Request #328 · StackStorm_ansible-st2](https://user-images.githubusercontent.com/1533818/209950790-b54cd2c9-c2e9-496f-9fe3-8b585a037dba.png)

The amount of versions here looks misleading, let's remove those and test only with one molecule version.
Similar to test-kitchen, as a developer/user who wants to modify or add Ansible tasks I don't care which version of Molecule we run as long as it runs the tests. Let's avoid overloading with this version info.

Reply:
I mean, one of the main reasons I chose to split the jobs between 3.6 and 3.8 was the Molecule version... would you rather just test both 3.6 and 3.8 against the same (lower) version of Molecule?

Reply:
Yes, just having one pinned version of Molecule is fine.